generator/src/main/java/com/scottlogic/deg/generator/walker/ReductiveDecisionTreeWalker.java

```diff
@@ -85,4 +98,8 @@ private void visualise(ConstraintNode rootNode, ReductiveState reductiveState){
             throw new RuntimeException(e);
         }
     }
+
+    public ReductiveDataGeneratorMonitor getMonitor(){
```

Comment:
Why does this exist, its a dependency - we shouldn't be exposing it?

Reply:
https://github.com/ScottLogic/data-engineering-generator/pull/374

have created this to deal with this issue